src/ui/MobileNavbar/NavItem.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import Svg from '@/ui/Svg/svelte'
+
+  export let title = ''
+  export let icon = ''
+  export let href = ''
+  export let active = false
+</script>
+
+<a class="btn column hv-center txt-m" {href} on:click class:active>
```

Comment:
1. Render a [`svelte:element`](https://svelte.dev/docs#template-syntax-svelte-element) since menu-button is a button element. Element's tag must specified using `tag` prop;
2. `href` prop declaration is redundant, since it can be passed using [`{...$$restProps}`](https://svelte.dev/docs#template-syntax-attributes-and-props)